WalletWasabi.Fluent/Views/Wallets/Send/TransactionPreviewView.axaml

```diff
@@ -49,7 +49,7 @@
 
       <Separator />
       <c:PreviewItem Icon="{StaticResource transceive_regular}"
-                     Text="via Bitcoin address">
+                     Text="To Bitcoin address">
```

Comment:
```suggestion
                     Text="to Bitcoin address">
```

should be lower case to flow in the whole sentence.

`to` is better than `via`.

Reply:
actually, it should not! 

All other lines start capitalized too.

Reply:
The initial plan was as @yahiheb described. But since then the things have changed... New rows added... Order changed.
I would say should be lower case if there is no separator between the two sections, otherwise upper.